packages/@tinacms/api-git/src/router.ts

```diff
@@ -49,6 +49,22 @@ export function router(repo: Repo, config: Partial<GitRouterConfig> = {}) {
     pushOnCommit,
   }: GitRouterConfig = { ...DEFAULT_OPTIONS, ...config }
 
+  if (!repo) {
+    /**
+     * Incase js users forget to pass in `repo`.
+     */
+    throw new Error(
+      '@tinacms/api-git#router(repo, config): Parameter `repo` is is missing.'
```

Comment:
```suggestion
      '@tinacms/api-git#router(repo, config): Parameter `repo` is missing.'
```

error messages are hard